code/controllers/subsystem/persistence.dm

```diff
@@ -5,8 +5,9 @@ var/datum/subsystem/persistence/SSpersistence
 	init_order = -100
 	flags = SS_NO_FIRE
 	var/savefile/secret_satchels
-	var/list/new_secret_satchels = list() //these are objects
-	var/list/old_secret_satchels = list() //these are associate
+	var/list/satchel_blacklist 		= list() //these are typecaches
```

Comment:
technically the list is one ENTIRE typecache (singular, like how it's a list not a lists) but I'm not anal enough to ask you to change this (but you should change the other comment so maybe bundle it as a 2for1 :P )